plugins/source/azure/policies/queries/account/locations_without_network_watchers.sql

```diff
@@ -7,7 +7,7 @@ select
   l.subscription_id,
   l.id,
   case
-    when anw.cq_id is null then 'fail' else 'pass'
+    when anw._cq_id is null then 'fail' else 'pass'
   end
-from azure_account_locations l
-  left join azure_network_watchers anw on l.name = anw.location
\ No newline at end of file
+from azure_subscriptions_locations l
```

Comment:
Should this be
```suggestion
from azure_subscription_locations l
```

Reply:
Unfortuately your suggestion is not the correct table name in the current state of the azure plugin.

Reply:
Thats what I figured. Thanks.